docs/side_quests/nf-core.md

```diff
@@ -0,0 +1,1427 @@
+# nf-core
+
+nf-core is a community effort to develop and maintain a curated set of analysis pipelines built using Nextflow.
```

Comment:
Totally open question- do you think we could use this opportunity to stress the independence of the nf-core community? 

```suggestion
nf-core is a community effort to develop and maintain a curated set of analysis pipelines built using Nextflow. The community is coordinated like X....
```